src/jsx.ts

```diff
@@ -25,6 +25,30 @@ export type FunctionComponent = (
   children?: VNode[]
 ) => VNode;
 
+export function Fragment(
```

Comment:
Any special reason why this is not lower case?

Reply:
I believe it's the convention in the community.

Please refer these documentations:
- https://www.typescriptlang.org/tsconfig#jsxFragmentFactory
- https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#pragmafrag

Reply:
I see, thanks